packages/fxa-content-server/server/lib/routes/react-app/route-definition-index.js

```diff
@@ -0,0 +1,195 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
```

Comment:
Note for reviewer(s) incl. me - this is extracted from `get-index`. Did a diffcheck and the only change is a required update to the path for `RELEASE` on L82

Reply:
Thanks, yeah, I even did a `git mv` for this file, but it's not showing up that way here.